D/DecFP/build_tarballs.jl

```diff
@@ -32,8 +32,12 @@ if [[ ${target} == *-w64-* ]]; then
     objext="obj"
 elif [[ ${target} == *-darwin* ]]; then
     HOST_OS="Darwin"
-    # clang causes failed tests on x86_64
-    CC="gcc"
+    # use clang instead of gcc to avoid https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/1339
+    CC="clang"
+    if [[ ${target} == x86_64* ]]; then
+        # avoid local test failure with clang -O1/2
+        CFLAGS_OPT+=" -O0"
```

Comment:
-O0 sounds pretty bad.

Reply:
If `clang` miscompiles things it would be better to either switch to a newer clang, or to gcc ?!? But you are switching away from gcc, so I am wondering what the reasoning is overall?